services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,11 @@
 ---
 title: "eth_call"
 ---
+import ParserOpenRPC from "@site/src/components/ParserOpenRPC"
+import { NETWORK_NAMES } from "@site/src/plugins/plugin-json-rpc"
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+<ParserOpenRPC
+  network={NETWORK_NAMES.linea}
+  method="eth_call"
```

Comment:
Whitespace in RHS div is off 
<img width="1361" alt="image" src="https://github.com/user-attachments/assets/8fc0c380-3752-417b-bb7e-6d521c8265ce">

Reply:
fixed with metadata